ExpressJS/utils/sequilize.js

```diff
@@ -0,0 +1,172 @@
+require('dotenv').config({ path: './utils/dev.env' });
+const fs = require('fs');
+const { Sequelize, Op, Model, DataTypes } = require('sequelize');
+
+
+const dbConnectionType = process.env.DB_CONNECTION_TYPE;
```

Comment:
fuck this console.log

Reply:
I guess are referring to the ones which logs the env-vars?

Reply:
Sorry about the words xD
What I mean is to just delete the console log of the env variables